src/servers/src/http.rs

```diff
@@ -58,6 +58,7 @@ use crate::server::Server;
 const HTTP_API_VERSION: &str = "v1";
 
 pub struct HttpServer {
+    timeout: u64,
```

Comment:
Using `std::time::Duration` instead.

Reply:
Yes, I did this before this mr, let me just revert to use Duration instead.